xml/System.Runtime.InteropServices.JavaScript/JSObject.xml

```diff
@@ -268,7 +268,7 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Returns <see langword="true" /> if the proxy was already disposed.</summary>
         <value>To be added.</value>
```

Comment:
```suggestion
        <value>
          <see langword="true" /> if the proxy was already disposed; otherwise, <see langword=
          
          false />.</value>
```

Reply:
Oops, had to fix this one.